pkg/kubelet/cm/memorymanager/memory_manager.go

```diff
@@ -91,6 +91,10 @@ type Manager interface {
 
 	// GetMemory returns the memory allocated by a container from NUMA nodes
 	GetMemory(podUID, containerName string) []state.Block
+
+	// GetExclusiveResources returns a list of resource names whose instances were esclusively
+	// allocated to this container.
+	GetExclusiveResources(pod *v1.Pod, container *v1.Container) []string
```

Comment:
I see, it's the same name here too. It should probably be its own interface then, rather than embedded in both the DeviceManager and MemoryManager interfaces.

Reply:
answering myself, because at least `GetDeviceRunContainerOptions` in device manager, extending 'Allocate' may be trickier than anticipated. It _seems_ that a getter is a simpler option overall.